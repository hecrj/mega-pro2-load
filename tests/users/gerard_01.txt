16
 
180
180
5
2000
70
60
50
40
30
20
10
80
100
100
130
150
 
10
 
8 5 4 6 0 0 9 0 0 0 1 7 0 0 2 3 0 0 10 0 0
 
10
3 5 6 7
 
20
2 14 2
 
45
4 11 13 15 9
 
37
1 8
 
50
3 10 14 16
 
50
2 12 1
 
80
1 5
 
1
5 1 2 3 5 6
 
1127
0
 
1127
0
 
 
-4 8
0
4 2 1 5 6
 
-1 4 1
 
-1 10 2
 
-5 0 2
 
-1 14 5
 
-1 14 6
 
-1 14 9
 
-3 10
 
-2 11
 
-5 0 5
 
-5 0 6
 
-4 9
1 4
0
 
-4 10
1 4
0
 
-1 4 10
 
-1 3 11
 
-4 1
0
1 6
 
-4 8
1 6
0
 
-1 6 14
 
-1 6 17
 
-6
