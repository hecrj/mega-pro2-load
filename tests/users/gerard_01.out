Peticion procesada y servidores
0 0
Peticion procesada y servidores
1 1
5
Pelicula mas solicitada
10 1
Peticion procesada y servidores
2 2
5
Peticion procesada y servidores
3 5
2
Peticion procesada y servidores
4 2
5
Servidores ocupados
2 3 1
5 4 1
Peticiones pendientes
0
Pelicula mas solicitada
10 1
Pelicula mas solicitada
14 2
Peticion procesada y servidores
5 2
9
Peticion procesada y servidores
6 5
8
Peticion procesada y servidores
7 0
Peticion procesada y servidores
8 60
8
